File: frc2018_3/src/main/java/ca/warp7/frc2018_4/auto/Baseline.java
package ca.warp7.frc2018_4.auto;

import ca.warp7.action.IAction;
import ca.warp7.frc2018_4.actions.DriveForTimeAction;

public class Baseline implements IAction.Mode {

    @Override
    public IAction getAction() {
        return new DriveForTimeAction(8, 0.2, 0.2);
    }
}

File: frc2018_5/src/main/kotlin/ca/warp7/frc2018/v5/auto/Baseline.kt
package ca.warp7.frc2018.v5.auto

import ca.warp7.action.IAction
import ca.warp7.action.impl.ActionMode
import ca.warp7.frc2018.v5.actions.DriveForTime

object Baseline : ActionMode() {
    override fun getAction(): IAction {
        return DriveForTime(0.5, 0.5, 2.0)
    }
}